src/toil/job.py

```diff
@@ -1005,6 +1005,9 @@ def _makeJobGraphs(self, jobGraph, jobStore):
             jobs = map(lambda successor:
                 successor._makeJobGraphs2(jobStore, jobsToJobGraphs), successors)
             jobGraph.stack.append(jobs)
+
+        #Write the jobs to the jobStore all at once
+        jobStore.batchCreate(jobsToJobGraphs.values())
```

Comment:
This serializes the job graphs, but they're still overwritten serially in _serialiseJobs below, right? I don't think the jobGraph command is set before this point, so the overwrite is still necessary as written. (Moving the lines which set .command up to here would probably work.)